cli/format/sort.go

```diff
@@ -34,3 +35,11 @@ func (s ByStartsAt) Swap(i, j int) { s[i], s[j] = s[j], s[i] }
 func (s ByStartsAt) Less(i, j int) bool {
 	return time.Time(*s[i].StartsAt).Before(time.Time(*s[j].StartsAt))
 }
+
+type ByAddress []*models.PeerStatus
+
+func (s ByAddress) Len() int      { return len(s) }
+func (s ByAddress) Swap(i, j int) { s[i], s[j] = s[j], s[i] }
+func (s ByAddress) Less(i, j int) bool {
+	return bytes.Compare([]byte(*s[i].Address), []byte(*s[j].Address)) < 0
```

Comment:
not sure why you cast string to bytes.

Reply:
I double-checked this logic and that was wrong. I need to collect an IP addresses from host:ip format. Then using net.IP to comparing using bytes will be much correct

Reply:
Here is the new way
``` 
func (s ByAddress) Less(i, j int) bool {
	ip1, _, _ := net.SplitHostPort(*s[i].Address)
	ip2, _, _ := net.SplitHostPort(*s[j].Address)
	return bytes.Compare(net.ParseIP(ip1), net.ParseIP(ip2)) < 0
}
```
Im not sure about the ignoring errors what do you think about this implementation

Reply:
However this still discards
127.0.0.1:1000
127.0.0.1:2000
127.0.0.1:3000

What do you think about this?
Do we sort by ip address if yes should we add ports as well ? or discards sorting ?